### PR TITLE
updated styles

### PR DESCRIPTION
**Pull request by PietroLiuzzo**
updated styles to meet team requirement and fix errors. I tested with the validator and it says all is good. let me know if there are issue. Thanks a lot!

### CODE REVIEW

**Comment by PietroLiuzzo** [comment]
Bildschirmfoto 2021-09-22 um 13 03 44
Sorry I cannot parse those error reports, but the validator says it is ok.

File: hiob-ludolf-centre-for-ethiopian-studies.csl [hunk 1]
```diff
@@ -20,9 +20,11 @@
     </contributor>
     <category citation-format="author-date"/>
     <category field="humanities"/>
-    <updated>2017-10-02T08:36:15+00:00</updated>
+    <updated>2021-09-22T12:29:15+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
+ <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
```
Comment (adam3smith) [inline hiob-ludolf-centre-for-ethiopian-studies.csl:26]: You're duplicating rights and `</info>` here, that's likely breaking everything else (you've also disabled allowing us to make changes to your pull request, which makes fixing things harder on our end)

Comment (PietroLiuzzo) [reply hiob-ludolf-centre-for-ethiopian-studies.csl:26]: I did not do this intentionally (disabling the changes) sorry about that.

**Comment by adam3smith** [comment]
Thanks! Could you run each style through https://formatter.citationstyles.org/ so the indenting is right? 

(I'd usually do this myself, but can't with the PR locked; IIRC that happened before with your pull request? Are you using any particular tool for this? Any chance you changes a default setting in GitHub? It's no big deal, but does mean we need to ask you to make small changes wed usually make ourselves before accepting)

**Comment by adam3smith** [comment]
Thanks!

**Comment by PietroLiuzzo** [comment]
thanks to you!

**Pull request merged**